scripts/exgfs_wave_init.sh

```diff
@@ -85,7 +81,8 @@ for grdID in "${grdALL[@]}"; do
     echo "INFO: mod_def.${grdID} succesfully created/copied"
   else
     echo "FATAL ERROR: No model definition file for grid ${grdID}"
-    err=3; export err; ${errchk}
+    export err=3
+    err_chk
```

Comment:
Delete the echo on line 83 (I was unable to add into commit suggestion) and move the message to the `err_chk`:
```suggestion
    export err=3
    err_chk "FATAL ERROR: No model definition file for grid ${grdID}"
```

Reply:
Fixed.